packages/rocketchat-livechat/client/views/app/livechatCurrentChats.js

```diff
@@ -59,6 +59,43 @@ Template.livechatCurrentChats.events({
 		instance.filter.set(filter);
 		instance.limit.set(20);
 	},
+	'click .delete-all'() {
```

Comment:
I suggest you ask the user who clicked the button to ensure that they really want to perform the action.
With this behaviour you can prevent accidental deletions of all chats.